vlib/v/checker/tests/eq_ne_op_wrong_type_err.vv

```diff
@@ -22,4 +24,15 @@ fn main() {
 	println([1] == arr)
 	println(arr != [0])
 	println([1] != arr)
+
+	arr_aaa := ArrAaa(arr)
+	println(arr_aaa == arr)
```

Comment:
tests should use `assert`

Reply:
this is check of type mismatch error for `==`. not result of `==`